scipy/stats/tests/test_continuous.py

```diff
@@ -701,7 +728,13 @@ def has_formula(order, kind):
         assert ref.shape == result_shape
         check(i, 'standardized', 'formula', ref,
               success=has_formula(i, 'standardized'))
-        check(i, 'standardized', 'general', ref, success=i <= 2)
+        if not (
+                isinstance(dist, Binomial)
+                and np.any((dist.n == 0) | (dist.p == 0) | (dist.p == 1))
+        ):
+            # This test will fail for degenerate case where binomial distribution
```

Comment:
Again, skip in point distribution case.

Reply:
https://github.com/mdhaber/scipy/pull/121/files#r2059159699